platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectImporter.java

```diff
@@ -194,9 +198,12 @@ public void importSources(FolderEntry baseFolder, String location, Map<String, S
         }
     }
 
-    private void cloneRepository(GitConnection git, String remoteName, String url, DtoFactory dtoFactory)
+    private void cloneRepository(GitConnection git, String remoteName, boolean recursiveEnabled, String url, DtoFactory dtoFactory)
```

Comment:
Why do we need send instance of DtoFactory into method?
Please put recursiveEnabled parameter to end of parameters list.